test/distributed/pipelining/model_registry.py

```diff
@@ -87,6 +87,12 @@ def forward(self, x):
         x = self.net2(x)
         return x
 
+    def init_weights(self):
+        torch.nn.init.xavier_uniform_(self.net1.weight)
+        torch.nn.init.xavier_uniform_(self.net2.weight)
+        torch.nn.init.zeros_(self.net1.bias)
+        torch.nn.init.zeros_(self.net2.bias)
```

Comment:
is this the place where model init happens?

Reply:
for the test case, yes.  But 'self' could be already an FSDP model, i just didn't test it that way